exotica_core/include/exotica_core/kinematic_tree.h

```diff
@@ -78,209 +74,202 @@ inline KinematicRequestFlags operator&(KinematicRequestFlags a, KinematicRequest
     return static_cast<KinematicRequestFlags>(static_cast<int>(a) & static_cast<int>(b));
 }
 
-class KinematicFrameRequest
+struct KinematicFrameRequest
 {
-public:
     KinematicFrameRequest();
-    KinematicFrameRequest(std::string frameALinkName, KDL::Frame frameAOffset = KDL::Frame(), std::string frameBLinkName = "", KDL::Frame frameBOffset = KDL::Frame());
-    std::string FrameALinkName;
-    KDL::Frame FrameAOffset;
-    std::string FrameBLinkName;
-    KDL::Frame FrameBOffset;
+    KinematicFrameRequest(std::string _frame_a_link_name, KDL::Frame _frame_a_offset = KDL::Frame(), std::string _frame_b_link_name = "", KDL::Frame _frame_b_offset = KDL::Frame());
+    std::string frame_a_link_name;
```

Comment:
`frame_A_link_name`, `frame_B_link_name` etc. - A and B are identifiers as used in papers etc. and should be kept capitalised as is (applies to everywhere)